src/resmom/mom_main.c

```diff
@@ -2780,67 +2781,65 @@ do_mom_action_script(int	ae,	/* index into action table */
 #ifdef	WIN32
 	shell = set_shell(pjob, pwdp);	/* machine dependent */
 
-	init_envp();
-
 	/*
 	 **	Setup environment
 	 */
 	/* UID */
 	sprintf(buf, "%d", pjob->ji_qs.ji_un.ji_momt.ji_exuid);
-	bld_wenv_variables("UID", buf);
+	env_array = bld_wenv_variables("UID", buf, env_array);
```

Comment:
check if bld_wenv_variables() can be deprecated in lieu of the mainstream bld_env_variables() function. Basically, we should try to reuse bld_env_variables() and modify that if it needs enhancement